lua/octo/util.lua

```diff
@@ -143,4 +144,9 @@ function M.update_reactions_at_cursor(bufnr, cursor, reactions, reaction_line)
   api.nvim_buf_set_var(bufnr, "reaction_line", reaction_line)
 end
 
+function M.format_date(to_format)
```

Comment:
should we rename `to_format` to `date`?

Reply:
not happy about `to_format` either, but `date` is already the lua-date import.
Maybe `date_string`? `iso8601`?

Reply:
`date_string` lgtm 👍🏼